src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -236,14 +236,15 @@ def node_test(
             stream=stream,
             credential_list=credential_list,
         ):
+            storage = DefaultRunStorage(base_dir=self.flow.code, sub_dir=Path(".promptflow/intermediate"))
```

Comment:
@lalala123123 please help to take a look

Reply:
LGTM